packages/size-limit/read-pkg-up.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path')
+const { promisify } = require('util')
+const fs = require('fs')
+
+const readFileAsync = promisify(fs.readFile)
+
+const readPkg = async cwd => {
```

Comment:
Use `let` for code style consistency

Reply:
Fixed in 84789586cb9d4aa93cb6945325a4d26933dce720